Poisson_surface_reconstruction_3/include/CGAL/Surface_mesher/Poisson_sphere_oracle_3.h

```diff
@@ -0,0 +1,464 @@
+// Copyright (c) 2006-2007  INRIA Sophia-Antipolis (France).
+// All rights reserved.
+//
+// This file is part of CGAL (www.cgal.org).
+//
+// $URL$
+// $Id$
+// SPDX-License-Identifier: GPL-3.0-or-later OR LicenseRef-Commercial
+//
+//
+// Author(s)     : Laurent RINEAU
+
+#ifndef CGAL_POISSON_SURFACE_MESHER_POISSON_SPHERE_ORACLE_3_H
+#define CGAL_POISSON_SURFACE_MESHER_POISSON_SPHERE_ORACLE_3_H
+
+#include <CGAL/license/Surface_mesher.h>
```

Comment:
TODO: fix copyright notice (date).

TODO: discussion about headers names (including path)

Plus the license-check header:
```suggestion
// Copyright (c) 2006-2007  INRIA Sophia-Antipolis (France).
// All rights reserved.
//
// This file is part of CGAL (www.cgal.org).
//
// $URL$
// $Id$
// SPDX-License-Identifier: GPL-3.0-or-later OR LicenseRef-Commercial
//
//
// Author(s)     : Laurent RINEAU

#ifndef CGAL_POISSON_SURFACE_MESHER_POISSON_SPHERE_ORACLE_3_H
#define CGAL_POISSON_SURFACE_MESHER_POISSON_SPHERE_ORACLE_3_H

#include <CGAL/license/Poisson_surface_reconstruction_3.h>
```